srv/StopBagRetention.srv

```diff
@@ -0,0 +1,9 @@
+## Request
```

Comment:
Given how generic this `.srv` is, why not just make it something like `BasicRequest.srv`?